src/App.js

```diff
@@ -35,10 +35,11 @@ const App = (props) => {
         insights.chrome?.globalFilterScope?.('insights');
         if (insights.chrome?.globalFilterScope) {
             insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
-                const [workloads, SID, selectedTags] = insights.chrome?.mapGlobalFilter?.(data, false, true);
+                let selectedTags = insights.chrome?.mapGlobalFilter?.(data)?.
+                filter(item =>  !item.includes('Workloads') || !item.includes('SAP ID (SID)')) || undefined;
                 batch(() => {
-                    dispatch(setWorkloads(workloads));
-                    dispatch(setSIDs(SID));
+                    dispatch(setWorkloads(data?.Workloads));
+                    dispatch(setSIDs(data['SAP ID (SID)']));
```

Comment:
Since we don't have the new way of mapping filter you might want to make this into an array
```JS
dispatch(setSIDs(Object.entries(data['SAP ID (SID)']).filter(([, { isSelected }]) => isSelected).map(([ key ]) => key)));
```